clap-v3-utils/src/input_validators.rs

```diff
@@ -387,25 +387,6 @@ where
     }
 }
 
-pub fn is_niceness_adjustment_valid<T>(value: T) -> Result<(), String>
-where
-    T: AsRef<str> + Display,
-{
-    let adjustment = value
-        .as_ref()
-        .parse::<i8>()
-        .map_err(|err| format!("error parsing niceness adjustment value '{value}': {err}"))?;
-    if solana_perf::thread::is_renice_allowed(adjustment) {
```

Comment:
lol... i had no idea this is how we tested for niceness. why not just `i8::from_str(s)...`? :upside_down_face: